packages/sysutils/busybox/scripts/init

```diff
@@ -153,6 +154,13 @@
       portable)
         SYSLINUX_DEFAULT="run"
         ;;
+      grub_live)
+        LIVE=yes
+        GRUB_DEFAULT="\"Live\""
+        ;;
+      grub_portable)
+        GRUB_DEFAULT="\"Run\""
```

Comment:
```
+        GRUB_DEFAULT="\"Live\""
+        GRUB_DEFAULT="\"Run\""
```
I know why you've got it like this, but it just looks... ugly.

At the risk of bikeshedding, my preference would be to double-quote the string in the update function rather than include the double quotes in the string itself.

Also, the `update_syslinux` function should be renamed `update_bootloader` now that it also handles grub.

Other than that it all looks fine.

Reply:
Ok no problem. But for the ```update_syslinux``` case, can I use other name instead? I saw another variable using same ```update_bootloader()``` in line 470. Maybe ```update_bootloaders``` ? Is that ok?

Reply:
Good point. Hmmm... having both `update_bootloader` and `update_bootloaders` could be confusing.

How about: `update_bootmenu`?

Reply:
Ok then I will proceed with ```update_bootmenu```.

Reply:
Ok everything done. :)